Infrastructure/AzureAdOptions.cs

```diff
@@ -0,0 +1,13 @@
+namespace WebApp_OpenIDConnect_DotNet.Infrastructure
```

Comment:
This should not be needed. it's already part of ASP.NET Core: See https://github.com/aspnet/AspNetCore/blob/master/src/Azure/AzureAD/Authentication.AzureAD.UI/src/AzureADOptions.cs
why did you add it?

Reply:
I was following convention with [IOptions ](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-2.2)dialog to be able to pull in additional details. Can be removed, but the point here is to have it inserted in different places via dependency injection to get required data from user settings perspective.
```
services.Configure<WebOptions>(configuration);
services.Configure<AzureAdOptions>(configuration.GetSection("AzureAd"));
```

Reply:
@bojanv : I'm still not clear why you don't use AzureADOptions (provided by AspNet Core)